### ISSUE TITLE
Support Hover Editor/New Window

### ISSUE THREAD

**Issue by aadimator** [juliang22/ObsidianTimestampNotes#15]
Can you kindly support opening the video in a Hover Editor or more preferrable, in the New Window option available in the Obsidian nowadays? That'd be very helpful, especially for those with dual monitors.

Thanks

**Comment by Kodema1** [comment]
Yes, having the video in full screen on a second monitor would be great!

**Comment by Kodema1** [comment]
oh I figured a workaround is to drag the text editor instead of the video to the second monitor basically does the same thing !

**Comment by cs-mshah** [comment]
There is a simple way - just right click on the video tab and click on convert to hover editor and you are done. All commands such as `ctrl + space` in the text note will have effects on the popped hover editor. Enjoy resizing this too.
